deployment/docker/README.md

```diff
@@ -1,5 +1,7 @@
 # Launching with Docker
 
+**DEPRECATED APRIL 2018 - SEE READ THE DOCS VERSION**
```

Comment:
We should delete the files that are obsoleted rather than having outdated information still appearing in the repo.

Reply:
Leave these in for now. After Read The Docs is working properly, I'll updated the README's to point to the documentation on Read The Docs.